### PR TITLE
js-json completed

### PR DESCRIPTION
**Pull request by rachaelkim831**
json

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    title: 'The Jekyll Island Chronicles (Book Three): A Last Call',
+    author: 'Steve Nedvidek'
+  }
+];
+
+console.log('typeof booksArray:', typeof booksArray);
+console.log('JSON.stringify:', JSON.stringify(booksArray));
+console.log('typeof JSON.stringify:', typeof JSON.stringify(booksArray));
+
+var string = {
```
Comment (Zhiwutian) [inline javascript-and-json/main.js:23]: This is not a JSON string. The big thing to remember is that you need single quotes on the outside of the string, and double quotes for all property names and values

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update to the JSON string

**Comment by rachaelkim831** [comment]
json

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**